examples/source/interactivity-dynamic-content/Subscription_Settings/index.html

```diff
@@ -0,0 +1,434 @@
+      <template type="amp-mustache">
+        <form action-xhr="<% base_path %>/subscription" method="post" id="form2"
+          on="submit-success: AMP.setState({ nextSubscription: null });
+            submit-error: AMP.setState({ nextSubscription: null });">
+          <input type="hidden" name="nextSubscription" [value]="nextSubscription">
+          <select
+            on="change: AMP.setState({ nextSubscription: event.value }), form2.submit;"
+            [disabled]="!!nextSubscription">
+            {{#options}}
+              {{#isSelected}}
+                <option value="{{value}}" [disabled]="!!nextSubscription" selected>
+                  {{text}}
+                </option>
+              {{/isSelected}}
+              {{^isSelected}}
+                <option value="{{value}}" [disabled]="!!nextSubscription">
+                  {{text}}
+                </option>
+              {{/isSelected}}
+            {{/options}}
+          </select>
+        </form>
+      </template>
+      <div placeholder>Loading...</div>
+      <div fallback>Something went wrong. Please refresh</div>
+    </amp-list>
+
+    <!--
+      Note that despite the `disabled` attribute being a [boolean attribute](https://www.w3.org/TR/2011/WD-html5-author-20110809/common-microsyntaxes.html#boolean-attribute), we can still effectively bind it because [`amp-bind` will remove the attribute when the binding expression is `false`](/documentation/components/amp-bind#bindings:~:text=Boolean%20expression%20results%20toggle%20boolean%20attributes.,false%2C%20the%20controls%20attribute%20is%20removed.).
+    -->
+
+    <!--
+      #### Step 5: Handling Form Submission Failure
+
+      When the form fails to submit we should display an error message and revert the `select` element to its previous value so that the user interface is consistent with the user's persisted subscription setting.
+
+      To display an error message when the form fails to submit we add element containing our error message to the `form` element's children, and annotate the child element with the [`submit-error` attribute](/documentation/components/amp-form#submit-error). The attribute ensures that the element is only visible after a failed form submission.
+
+      Reverting the `select` element's value to its previous value requires fully managing the `select` element's value by binding the `selected` attribute of each of its `option` elements. This requires maintaining the `select` element's current value in a `currentSubscription` state variable and the element's previous value in a `previousSubscription` state variable.
+
+      When the `select` element's value is changed we set `currentSubscription` to the new value and `previousSubscription` to the element's previous value, which is stored in the `currentSubscription` state variable if this is not the first time the element's value has been changed or in the `currentSubscription` field of the `amp-list` response otherwise.
```

Comment:
```suggestion
      When the `select` element's value changes, we set `currentSubscription` to the new value and `previousSubscription` to the element's previous value. We store these in the `currentSubscription` state variable, if this is not the first time the element's value has changed. Otherwise, it's stored in the `currentSubscription` field of the `amp-list` response.
```